javascript/webhooks-nodejs/README.md

```diff
@@ -28,31 +28,47 @@ Ngrok is used to create a publicly accessible "tunnel" (i.e., URL) to a port on
 
 ## Usage
 
-1. Create a file named **.env** in the root directory of this app. In the file, include the following line:
+After ngrok is successfully running, the next steps are to run the server (**index.js**), then establish a webhook using the provided webhook setup script (**createWebhook.js**).
+
+1. Create a file named **.env** in the root directory of this app. In the file, include the following lines:
 
 ```
 X_HOOK_SECRET=
+PAT=
 ```
 
-The Express server writes the `X-Hook-Secret` obtained from the [webhook handshake](https://developers.asana.com/docs/webhooks-guide#the-webhook-handshake) directly into this local file, allowing the `X-Hook-Secret` to persist across server restarts. As such, this server is designed to handle a single `X_HOOK_SECRET`, which is associated with a single webhook.
+You do not need to manually supply a value for `X_HOOK_SECRET`. The Express server writes the `X-Hook-Secret` obtained from the [webhook handshake](https://developers.asana.com/docs/webhooks-guide#the-webhook-handshake) directly into this local file, allowing the `X-Hook-Secret` to persist across server restarts. As such, this server is designed to handle a single `X_HOOK_SECRET`, which is associated with a single webhook.
 
 > **Note:** In a production environment, this value should be securely stored in a database. For security reasons, ensure that you never commit or expose this value in a public repository.
 
-For your convenience, a sample **.env.template** file is included in the root directory of this application.
+Enter your [personal access token](https://developers.asana.com/docs/personal-access-token) for `PAT`. For your convenience, a sample **.env.template** file is included in the root directory of this application.
 
 2. Start the webhook server:
 
 ```
 npm run dev
 ```
 
-3. Create a webhook by calling `POST /webhooks` and specifying given resource (e.g., a task). Feel free to use the [Postman Collection](https://developers.asana.com/docs/using-postman) or the [POST /webhooks](https://developers.asana.com/reference/createwebhook) page in the API Explorer to make your request.
+3. Edit the **createWebhook.js** setup script to include details for your [POST /webhooks](https://developers.asana.com/reference/createwebhook) request:
+
+```js
+// TODO: Replace these values with your target URI, object ID, filter, and resource type
+const targetUri = "https://your.url.here/receiveWebhook"; // The webhook server's public endpoint for receiving webhooks
```

Comment:
Nit: `your.url.here` can we use `<>` convention here it makes it more clear and it's more common in our docs with the client libraries.

TLDR; `your.url.here` -> `<YOUR_URL_HERE>`

Reply:
Fixed!

Reply:
It looks like you fixed this in the `createWebhook.js` file but not the `README.md`